addon/-private/external/task-decorators.js

```diff
@@ -23,6 +23,7 @@ function taskFromPropertyDescriptor(
   let tasks = new WeakMap();
   let options = params[0] || {};
   let factory = new factoryClass(key, taskFn, options);
+  factory._setupEmberKVO(target);
```

Comment:
I think the idea was to keep anything ember-specific out of the `external` module, but if we're not doing the separate Ember descriptors, this seems reasonable for now, especially if we're planning on getting rid of observer support. We can also figure out a refactor later if we need to.

Reply:
👍  Yeah I think we can revisit later